pkg/apis/core/annotation_key_constants.go

```diff
@@ -106,4 +106,12 @@ const (
 	// This annotation is used by the Attach Detach Controller to determine whether to use the in-tree or
 	// CSI Backend for a volume plugin on a specific node.
 	MigratedPluginsAnnotationKey = "storage.alpha.kubernetes.io/migrated-plugins"
+
+	// PodDeletionCost can be used to set to an int64 that represent the cost of deleting a
+	// pod compared to other pods belonging to the same ReplicaSet. Pods with lower deletion
+	// cost are preferred to be deleted before pods with higher pod-deletion-cost.
```

Comment:
what is the implicit cost of a pod that does not specify the annotation? can a negative value be specified?

Reply:
mirror doc updates here out to staging/src/k8s.io/api/core/v1/annotation_key_constants.go as well